support-frontend/assets/pages/contributions-landing/components/ContributionAmountChoices.jsx

```diff
@@ -0,0 +1,175 @@
+// @flow
+
+// ----- Imports ----- //
+import React from 'react';
+import { type SelectedAmounts } from 'helpers/contributions';
+import {
+  type Amount,
+  type ContributionType,
+} from 'helpers/contributions';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import {
+  type IsoCurrency,
+  currencies,
+  spokenCurrencies,
+} from 'helpers/internationalisation/currency';
+import { formatAmount } from 'helpers/checkouts';
+import { ChoiceCardGroup, ChoiceCard } from '@guardian/src-choice-card';
+import ContributionAmountChoicesChoiceLabel from './ContributionAmountChoicesChoiceLabel';
+import { from, until } from '@guardian/src-foundations/mq';
+import { css } from '@emotion/core';
+
+const choiceCardGroupOverrides = css`
+  > div {
+    ${until.leftCol} {
+      flex-wrap: wrap;
+    }
+    margin-top: -8px;
+  }
+
+  > div > label {
+    ${from.tablet} {
+      max-width: 100px;
+    }
+    margin-top: 8px !important;
+  }
+`;
+
+const choiceCardGrid = css`
+  ${from.mobileLandscape} {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 8px;
+  }
+`;
+
+type ContributionAmountChoicesProps = {|
+  countryGroupId: CountryGroupId,
+  currency: IsoCurrency,
+  contributionType: ContributionType,
+  validAmounts: Amount[],
+  showOther: boolean,
+  selectedAmounts: SelectedAmounts,
+  selectAmount: (
+    Amount | "other",
```

Comment:
I think to be consistent, `"other"` should use single-quotes.

Reply:
I've been using a combo of prettier and eslint (maybe I shouldn't be using prettier? but I do really like it...) and this seems the formatting it gives me (if I change to single quotes it reverts on save)